test/integrated-devnet-tests/predeployed-accounts-test/check.ts

```diff
@@ -0,0 +1,8 @@
+import { hardhatStarknetTest } from "../../utils/cli-functions";
+import { checkDevnetIsNotRunning } from "../../utils/utils";
+
```

Comment:
Add a comment here describing what we are testing and why this is in integrated-devnet tests.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/374/commits/e7b25f93f7cf0ee5428958cf1b3e7469586a624d